sdk_container/src/third_party/coreos-overlay/app-emulation/containerd/files/containerd.service

```
@@ -3,7 +3,7 @@ Description=Containerd Container Daemon
 Documentation=http://github.com/docker/containerd
```

```suggestion
Documentation=https://containerd.io
After=network.target local-fs.target
```
and `Description=containerd container runtime`

---

Yes, maybe follow the upstream order of settings for an easier to understand `diff` output

---

Thanks, looks good